spec.yaml

```diff
@@ -109,6 +109,51 @@ paths:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
 
   /v4/clusters/:
+    get:
+      operationId: getClusters
+      tags:
+        - clusters
+      summary: Get clusters
+      description: |
+        This operation allows to fetch a list of clusters.
```

Comment:
Hmm. I pretty much copied myself from other descriptions. Let's ask @rossf7 or @JosephSalisbury for native speaker help. Is there a problem with that phrase? If yes, what's a better way?